### DISCUSSION TITLE
pydantic model with generics isn't supported

### DISCUSSION THREAD

**Discussion by bugzpodder** [strawberry-graphql/strawberry#1678] [Q&A] [answered]
```
class BoolFieldFilter(FieldFilter[bool]):
    pass
```
It works if you subclass the generic class (`BoolFieldFilter`) but not directly `FieldFilter[bool]`

Moved to issue: https://github.com/strawberry-graphql/strawberry/issues/1680

**Comment by patrick91** [comment, marked as answer]
hi @bugzpodder, I think this should be an issue more than a discussion, same for https://github.com/strawberry-graphql/strawberry/discussions/1676 😊
